frontend/src/components/worksheets/NewRun/NewRun.jsx

```diff
@@ -300,6 +301,14 @@ class NewRun extends React.Component<
 
         if (command) args.push(command);
 
+        // exclude_patterns a list of arguments, so put it at the end of commands
+        if (exclude_patterns) {
+            args.push(`--exclude-patterns`);
+            for (let i = 0; i < exclude_patterns.length; i++) {
```

Comment:
We can just replace this with

```
args.push(...exclude_patterns);
```

See https://stackoverflow.com/questions/1374126/how-to-extend-an-existing-javascript-array-with-another-array-without-creating